src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResource.kt

```diff
@@ -137,6 +138,30 @@ class UserResource(
   ): StaffDetail =
     userService.findByStaffId(staffId)
 
+  @PreAuthorize("hasAnyRole('ROLE_USE_OF_FORCE', 'MANAGE_NOMIS_USER_ACCOUNT')")
```

Comment:
inconsistent here - one has the `ROLE_` prefix and the other hasn't.  Not sure if you need the prefix for the `PreAuthorize`.

Reply:
as suggested, removed MANAGE_NOMIS_USER_ACCOUNT